packages/website/src/features/Deploy/TransactionDisplay.tsx

```diff
@@ -285,6 +285,8 @@ export function TransactionDisplay(props: {
                 pkgUrl={
                   hintData.isSinglePackage ? hintData.cannonPackage : pkgUrl
                 }
+                cannonInfo={cannonInfo}
+                isPreloaded={hintData.isSinglePackage}
```

Comment:
In the case that is not a single package, it would still have performance issues?

Reply:
yes 😞 , I understand this is not too common though (to queue a lot of txs from different packages), but we can let it that way and think in another solution for the multipackage, something we can do is to fetch the data for unique packages and then pass the information to the corresponding txs, I mean, to fetch just the necessary data and not fetch data for each txn